analyses/tp53_nf1_score/run_classifier.sh

```diff
@@ -12,20 +12,24 @@
 set -e
 set -o pipefail
 
+# If running for subtyping, use the base histologies file
 RUN_FOR_SUBTYPING=${OPENPBTA_BASE_SUBTYPING:-0}
 
+# If we're running for figure generation, set to 1
```

Comment:
@sjspielman does this approach (more details on #1394) work for you?

Reply:
:100: